docs/UserGuideNew.md

```diff
@@ -0,0 +1,276 @@
+
+
+## 2. Quick Start
+
+1. Ensure you have Java `11` or above installed on your computer. You can download Java 11 via this [link.](https://www.oracle.com/sg/java/technologies/javase-jdk11-downloads.html)
+   * Set-up tutorial:
+        * [Windows user](https://java.tutorials24x7.com/blog/how-to-install-java-11-on-windows)
+        * [Mac user](https://docs.oracle.com/en/java/javase/11/install/installation-jdk-macos.html#GUID-2FE451B0-9572-4E38-A1A5-568B77B146DE)
+        * [Linux user](https://docs.oracle.com/en/java/javase/11/install/installation-jdk-linux-platforms.html#GUID-737A84E4-2EFF-4D38-8E60-3E29D1B884B8)
+   
+2. Download the latest `remindMe.jar` from our [GitHub release page](https://github.com/AY2021S2-CS2103T-W15-1/tp/releases).
+Copy the file to the folder you want to use as the home folder for your RemindMe.
+Double click the file to start the app. 
+   
+3. Alternatively, you can run the command line java -jar RemindMe.jar in your terminal to start the application.
+Type the command in the command box and press Enter to execute it.
+The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+   ![Ui](images/Ui.png)
+
+4. Press start to proceed to the main window.
+   ![UiMainWindow](images/UiMainWindow.png)      
+
+5. Type the command in the command box and press enter on the keyboard to execute it. <br> eg. typing `help` and press enter will open the help window.
+
+6. Some example commands you can try:
+   * **`add m/CS2103`** : Adds a module name CS2103.
+     
+   * **`calendar`**: Dsiplays the calendar with the tasks' deadlines and friends' birthdays.
+     
+   * **`exit`** : Exits the app.
+   
+7. Refer to [Section 3: Features](#3-features) for details of each command.  
+
+
+--------------------------------------------------------------------------------------------------------------------
+
+## 3. Features
+
+<div markdown="block" class="alert alert-info">
+
+**Notes about the command format:**<br>
+
+* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
+  e.g. in `add m/MODULE`, `MODULE` is a parameter which can be used as `add m/CS2103`.
+
+
+* Items in square brackets are optional.<br>
+  e.g `add n/Name [t/TAG]` can be used as `add n/Alice t/friends` or as `add n/Alice`.
+  
+
+* Extraneous parameters for commands that do not take in parameters (such as `help` and `exit`) will be ignored.<br>
+  e.g. if the command specifies `help 123`, it will be interpreted as `help`.
+
+</div>
+
+
+### 3.1 Viewing Help : `help`
+
+Shows a help page.
+
+![help message](images/helpMessage.png)
+
+Format: `help`
+
+### 3.2 Adding: `add`
+This section show features that deals with the adding of entries in the RemindMe app.
+<br>
+<br>
+#### 3.2.1 Adding a person
+Add a person with his/her birthday to the RemindMe app.
+
+* Note: Duplicate person(person with the same name) is not allowed. 
```

Comment:
Some notes are bold, while others ain't. I think it would be better if it was consistent